src/Microsoft.AspNet.OData.Shared/DeltaOfTStructuralType.cs

```diff
@@ -237,9 +314,9 @@ public override bool TryGetPropertyValue(string name, out object value)
         /// The nested Property type will be <see cref="IDelta"/> of its defined type.
         /// </remarks>
         /// </summary>
-        /// <param name="name">The name of the nested Property</param>
-        /// <param name="value">The value of the nested Property</param>
-        /// <returns><c>True</c> if the Property was found and is a nested Property</returns>
+        /// <param name="name">The name of the nested Property.</param>
+        /// <param name="value">The value of the nested Property.</param>
+        /// <returns><c>True</c> if the Property was found and is a nested Property.</returns>
```

Comment:
```suggestion
        /// <param name="value">The value of the nested property.</param>
        /// <returns>true if the property was found and is a nested property; otherwise false.</returns>
```